### DISCUSSION TITLE
Get the comment of the review step as env variable in the pipeline

### DISCUSSION THREAD

**Discussion by mohalsherif** [community/community#38344] [Actions]
When we add an environment in a pipeline job and the environment has required reviewers, then we get to have a manual step in the pipeline. In this manual step, one could enter a comment as well, during accepting it, as in this [link](https://docs.github.com/en/actions/managing-workflow-runs/reviewing-deployments#approving-or-rejecting-a-job). 
Is there a way to access this comment in the pipeline as Environment variable, so we can use it (i.e. send an email with it for example)?

**Comment by ossanna16** [comment]
Hi there @mohalsherif and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
